ert3/workspace/_workspace.py

```diff
@@ -227,6 +227,34 @@ def _validate_resources(self, ensemble_config: ert3.config.EnsembleConfig) -> No
                         f"Resource must be a regular file: '{resource.source_location}'"
                     )
 
+    def suggest_local_run_path(
+        self, basename: str = "local-test-run", run_id: str = "abcdef"
+    ) -> Path:
+        """Suggest an absolute directory path that can be used for a local test
+        run.
+
+        The returned path is guaranteed not to exist.
+
+        Args:
+            basename: This will be part of returned directory name, with
+                an index appended.
+            runid: An id that will always be included in the suggested runpath
+
+        Returns:
+            Non-existing path
+        """
+        suggestion = Path(self._path.absolute() / (basename + f"-{run_id}"))
```

Comment:
Why not just `basename + f"-str(uuid.uuid4())"`, instead of checking every +1 ?

Reply:
I think 6 digit hex looks less scary for users compared to `uuid4()`. This is used to make directories that users should `cd` into.

Reply:
```python
>>> uuid.uuid4().hex
'fd1d4c1f25b944749b11480f36622dfe'
```

Reply:
What about we print out a user manual for inspecting the output? That manual would include 

```bash
$ cd local-test-run-fd1d4c1f25b944749b11480f36622dfe

# You are now free to look around and inspect files…
```

Reply:
Use `uuid` just because its more conventional for unique resources.

Reply:
ok!